data/json/recipes/recipe_deconstruction.json

```diff
@@ -2837,6 +3010,21 @@
     "qualities": [ { "id": "SCREW", "level": 1 }, { "id": "WRENCH", "level": 1 }, { "id": "SAW_M", "level": 1 } ],
     "components": [ [ [ "motor_tiny", 3 ] ], [ [ "processor", 3 ] ], [ [ "RAM", 2 ] ], [ [ "plastic_chunk", 2 ] ] ]
   },
+  {
+    "result": "sewing_kit",
+    "type": "uncraft",
+    "time": "30 s",
+    "components": [ [ [ "scrap", 2 ] ], [ [ "plastic_chunk", 1 ] ] ]
+  },
+  {
+    "result": "steel_pan",
+    "type": "uncraft",
+    "skill_used": "fabrication",
+    "difficulty": 1,
+    "time": "1 m 30 s",
+    "qualities": [ { "id": "HAMMER", "level": 1 } ],
+    "components": [ [ [ "scrap", 10 ] ] ]
```

Comment:
200ml/250ml of scrap metal.

Reply:
Sorry, I will make the same as the crafting recipe.